### PR TITLE
plugingetter: Avoid all caps for constant declarations

### PR DESCRIPTION
**Pull request by aaronlehmann**
Go style calls for mixed caps instead of all caps:
https://golang.org/doc/effective_go.html#mixed-caps

Change `LOOKUP`, `ACQUIRE`, and `RELEASE` to `Lookup`, `Acquire`, and `Release`.

I will rebase this after #29556 is merged.

Note that vendoring will be an issue because of cyclic dependencies between repositories. We should probably vendor a fork of libnetwork, and then upstream the change.

### CODE REVIEW

**Comment by tonistiigi** [comment]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸
@aaronlehmann needs a rebase and vendoring 👼

**Comment by aaronlehmann** [comment]
Given the cyclic dependency issue, vendoring will have to be a two-stage effort. Are people okay with the original plan of vendoring a libnetwork fork in this PR, and then upstreaming the change once this is merged?

**Comment by vdemeester** [comment]
@aaronlehmann I'm sad we have cyclic dependency issues (libnetwork and swarmkit at least right ?) and we might want to work to sort this out. But this should not block this PR so I'm fine with vendoring a libnetwork fork in that PR and have a follow-up. Could you update `vendor.conf` to make this ? 👼

**Comment by cpuguy83** [comment]
Might be better for libnetwork to just copy rather than deal with these cyclic issues, no?
I can't see this ever changing.

**Comment by aaronlehmann** [comment]
Rebased and updated to vendor a temporary libnetwork fork.

**Comment by stevvooe** [comment]
LGTM

How is this used in `libnetwork`?

**Comment by mavenugo** [comment]
@stevvooe libnetwork ultimately calls the driver APIs for any CNM events such as add-network, create-endpoint, etc... so it uses the plugin API to get the remote client to make that API invocation.

@aaronlehmann @cpuguy83 I think it is better to keep the `Get` API free of `Acquire`, `Release` and `Lookup` primitives and make these internal implementation of plugin package. And let the plugin users to call simple APIs to achieve the same functionality. Having `Acquire` or `Release` flags as part of Get api isnt very intuitive. Also having explicit functions will avoid the cyclic dep issue (which is a plus).

**Comment by stevvooe** [comment]
@mavenugo That is pretty unfortunate.

Could we get these refactored so `libnetwork` just takes some interface and the reference counting sillyness is unexported?

Either way, let's merge this.

**Comment by vdemeester** [comment]
@aaronlehmann I'll let you update and re-vendor 👼

**Pull request merged**
